src/components/Header.js

```diff
@@ -1,10 +1,70 @@
-import React from 'react';
+import React, { useState } from 'react';
 import styled from 'styled-components';
 
-export default function Header() {
-	return <Wrapper>발신자 변경해주는 상단바</Wrapper>;
+export default function Header(props) {
+	// 프로필 누구인지
+	const [person, setPerson] = useState(true);
+	
+	const inputTagStyle = {		// 메시지 입력창
+		display: 'flex',
+		height: '100%',
+		width: '70px',
+		borderRadius: '30%',
+	}
+	
+	const showWhoActiveTextStyle = {	// "OO 현재 활동중"
```

Comment:
주석에 의존하여 변수/함수 네이밍과 코드 구조를 소홀히 하는 것은 나쁜 습관이랍니다 ㅜㅜ
주석은 가급적 쓰지 말고, 코드를 통해서 드러날 수 있게 노력해주세요!
스타일 객체는 명사형으로, 역할을 명확하고 자세하게 적어주세요.

동사형보단 여기서는 명사형이 더 맞는 것 같습니다.
```suggestion
	const currentUserIndicatorStyle = {
```